src/cpp/library/CUDA/cuda_runtime_api.h

```diff
@@ -50,6 +51,37 @@ cudaError_t cudaMemcpy(T1 dst, T2 src, size_t count, enum cudaMemcpyKind kind)
   return __cudaMemcpy(dst, src, count, kind);
 }
 
+cudaError_t cudaMalloc(void **devPtr, size_t size)
```

Comment:
As this function is defined in a header file, it will lead to a multiple definitions error in case 2 different translation units include this file. You might want to either make it inline or instead of a header to define it in a .c file. Same applies to all non-inline non-static function definitions in headers.